manifests/exporter.pp

```diff
@@ -5,13 +5,28 @@
 #   include rsan::exporter
 class rsan::exporter {
 
-
-
 ########################1.  Export Logging Function######################
 # Need to determine automatically the Network Fact IP for the RSAN::importer node automatically, applies to all infrastructure nodes
 #########################################################################
 
-
+class { '::nfs':
```

Comment:
This should be indented 2 more spaces than the class declaration since we're inside the curly braces if the class itself. (The same as the resources below)